### DISCUSSION TITLE
Tailwind psudo elements

### DISCUSSION THREAD

**Discussion by hsheikhali1** [tailwindlabs/tailwindcss#7499] [Help]
So, I have a question regarding how to use psudo-elements in tailwind. It shows in the docs that I just need to use the right prefix but when I try `after:content-['*']` it doesn't work at all ? Am i missing something?

**Comment by NovusOpus** [comment]
Can your provide a minimal reproduction or code example? , it's hard to know what could be the issue if you don't provide any code or example to look at.

But here's a working example in: [Tailwind Play](https://play.tailwindcss.com/CTFbMMidOK)

**Reply by hsheikhali1** [reply]
So I literally copied and pasted your code and it didn't work.. 
![Screen Shot 2022-02-17 at 11 45 32 AM](https://user-images.githubusercontent.com/14851850/154529437-e7bf315a-2cbf-4648-82f8-337dea1a9711.png)

PS, don't mind the position of the elements.. I just pasted that in a project I am working on atm

**Reply by hsheikhali1** [reply]
![Screen Shot 2022-02-17 at 11 50 18 AM](https://user-images.githubusercontent.com/14851850/154530289-26155799-4dd9-450c-984d-b53461c209f6.png)

That's a better screenshot.
![Screen Shot 2022-02-17 at 11 48 47 AM](https://user-images.githubusercontent.com/14851850/154530461-66a1c028-7581-43ab-a3ae-3e5bbe247cf6.png)
.

**Reply by NovusOpus** [reply]
Sadly this doesn't help too much since we cant see the full picture of what could be causing this just by seeing at a screenshot, we know the pseudo-elements are working as you can see in the Tailwind Play example in my reply above, so there's something going on from your side.

I understand if you cant share actual code from your work/project so it's fine, but maybe create a minimal reproduction example that contains no code from your actual work or project but can recreate the problem? 

In the mean time here's a few things that could be causing this:

1. Before and After pseudo-elements are only available with [Just-in-Time mode](https://tailwindcss.com/docs/just-in-time-mode) enabled on Tailwind CSS version 2.2+, and they available/enabled by default with Tailwind CSS version 3.0+.

2. Make sure the path containing your components or templates files(HTML, JS, JSX, TS, TSX ...) where you're using Tailwind Classes or Utilities are listed in your[ `content`](https://tailwindcss.com/docs/content-configuration) configuration in your `tailwind.config.js`

**Reply by sntran** [reply]
You're using React with `className` instead, so check that React outputs the class names correctly. Separators like colon (`:`) may need to be escaped.

**Reply by hsheikhali1** [reply]
@sntran Yes, className does output the correct classes. @NovusOpus I understand - I actually started tinkering in the docs and found that I have to have `jit` enabled to get it working. So yes, you're right! But I think it should be noted in the docs? Or maybe it is but I missed it..

Anyway, it's working now :) Thanks for the help regardless
